django/db/backends/mysql/operations.py

```diff
@@ -64,6 +68,12 @@ def datetime_trunc_sql(self, lookup_type, field_name, tzname):
         fields = ['year', 'month', 'day', 'hour', 'minute', 'second']
         format = ('%%Y-', '%%m', '-%%d', ' %%H:', '%%i', ':%%s')  # Use double percents to escape.
         format_def = ('0000-', '01', '-01', ' 00:', '00', ':00')
+        if lookup_type == 'quarter':
+            fmt = (
+                "CAST(DATE_FORMAT(MAKEDATE(YEAR({}), 1) + INTERVAL QUARTER({}) QUARTER - " +
+                "INTERVAL 1 QUARTER, '{}') AS DATETIME)"
+            )
+            return fmt.format(field_name, field_name, '%%Y-%%m-01 00:00:00')
```

Comment:
Put the format string inside `fmt` because it's not really a parameter since it's always the same thing?